src/services/web3/web3.plugin.ts

```diff
@@ -105,6 +106,9 @@ export default {
         rpcProviderService.getJsonProvider(chainId.value)
     );
     const signer = computed(() => pluginState.connector?.provider?.getSigner());
+    const web3Provider = computed(() => {
+      return new Web3Provider(pluginState.connector.provider as any);
+    });
```

Comment:
Lets rename this to be more explicit.
```suggestion
    const userProvider = computed(() => {
      return new Web3Provider(pluginState.connector.provider as any);
    });
```